lib/rules/web/admin_console/4.4/monitoring_metrics.xyaml

```diff
@@ -0,0 +1,139 @@
+click_primary_menu_developer_mode:
```

Comment:
`click_secondary_menu`

Reply:
The primary menu for develop mode is different from admin mode
`When I run the :goto_monitoring web action                         # features/step_definitions/web.rb:13
      [03:25:00] INFO> running web action goto_monitoring ... 
      [03:25:00] INFO> params..
      [03:25:00] INFO> running web action expand_primary_menu ... 
      [03:25:00] INFO> elements..
      [03:25:01] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:02] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:03] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:04] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:05] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:06] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:07] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:08] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:09] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:10] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:11] INFO> found 0 element elements with selector: {:xpath=>"//*[contains(@data-component, 'pf-nav-expandable') and contains(text(), 'Monitoring')]"}
      [03:25:11] INFO> embedding "2020-03-23T11:25:11+08:00-screenshot.png"
      [03:25:11] INFO> embedding "2020-03-23T11:25:11+08:00.html"
    Then the step should succeed                                       # features/step_definitions/common.rb:4
      the step failed (RuntimeError)`